contributors/guide/README.md

```diff
@@ -247,7 +247,7 @@ If you're looking to run e2e tests on your own infrastructure, [kubetest](https:
 ## Issues Management or Triage
 
 Have you ever noticed the total number of [open issues](https://issues.k8s.io)?
-Helping to manage or triage these open issues can be a great contributionand a great opportunity to learn about the various areas of the project.
+Helping to manage or triage(decide the order) these open issues can be a great contribution and a great opportunity to learn about the various areas of the project.
```

Comment:
I'm not sure what "decide the order" is supposed to mean here. This might need to be reworded to be more clear.

Also should have a space before the opening "(".

Reply:
decide the order means that its helps to decide in which order the issues needs to be managed.

Reply:
That is a useful thing to do, but I still think this needs to be reworded to be more clear.

Reply:
Ok I will rewrite to something more clear.